bitnami/contour/templates/envoy/hpa.yaml

```
@@ -0,0 +1,35 @@
+{{- if and .Values.envoy.autoscaling.enabled (eq .Values.envoy.kind "Deployment") }}
```

Also check that Envoy is enabled:
```suggestion
{{- if and .Values.envoy.enabled .Values.envoy.autoscaling.enabled (eq .Values.envoy.kind "Deployment") }}
```